.github/workflows/ci.yml

```diff
@@ -486,7 +486,9 @@ jobs:
           github.ref == 'refs/heads/main')
         run: |-
           gsutil -h "Cache-Control: public, max-age=3600" cp ./target/release/*.zip gs://dl.deno.land/canary/$(git rev-parse HEAD)/
-          echo ${{ github.sha }} > canary-latest.txt
+          lastest_canary_hash=$(gsutil cat gs://dl.deno.land/canary-$(rustc -vV | sed -n "s|host: ||p")-latest.txt) && \
+          git cat-file -e $(lastest_canary_hash) && \
```

Comment:
I guess this will error and fail the step? Probably ok, but might lead to main looking more red than it is?

Reply:
Improved a bit by adding a conditional